drivers/storage/rackspace/executor/rackspace_executor.go

```diff
@@ -0,0 +1,354 @@
+package executor
+
+// // StorageExecutor is the part of a storage driver that is downloaded at
```

Comment:
Hi @akiraw,

Please remove the header comments here. I know why you included them while doing dev, but they're no longer needed. Thanks!